### PR TITLE
Use common `sparseF32Range` instead of defining it multiple different places

### PR DESCRIPTION
**Pull request by zoddicus**
<hr>

**Requirements for PR author:**

- [x] All missing test coverage is tracked with "TODO" or `.unimplemented()`.
- [x] New helpers are `/** documented */` and new helper files are found in `helper_index.txt`.
- [x] Test behaves as expected in a WebGPU implementation. (If not passing, explain above.)

**Requirements for [reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md):**

- [ ] Tests are properly located in the test tree.
- [ ] [Test descriptions](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) allow a reader to "read only the test plans and evaluate coverage completeness", and accurately reflect the test code.
- [ ] Tests provide complete coverage (including validation control cases). **Missing coverage MUST be covered by TODOs.**
- [ ] Helpers and types promote readability and maintainability.

When landing this PR, be sure to make any necessary issue status updates.

### CODE REVIEW

**Comment by zoddicus** [comment]
This depends on https://github.com/gpuweb/cts/pull/1748, so the only new code is in https://github.com/gpuweb/cts/pull/1754/commits/e6506c06108239a3842dac49ef3a6a9b49f16721

**Review by dj2 — APPROVED** [review]

**Review by ben-clayton — CHANGES_REQUESTED** [review]
No PR description, and the PR subject doesn't seem to match the content.

**Comment by zoddicus** [comment]
This is based off my `dot` implementation, so all of the new code is in patch 59c12e0fe6b28ed32933022831501eb479431c7f

**Comment by zoddicus** [comment]
PTAL, this is ready for review

**Comment by zoddicus** [comment]
@ben-clayton can you take another look at this, GitHub is wanting your approval to land, since you requested changes.

File: src/webgpu/util/math.ts [hunk 1]
```diff
@@ -419,12 +419,16 @@ export function biasedRange(a: number, b: number, num_steps: number): Array<numb
 }
 
 /**
- * @returns an ascending sorted array of numbers spread over the entire range of 32-bit floats
+ * @returns a ascending sorted array of numbers spread over the entire range of 32-bit floats
```
Comment (ben-clayton) [inline src/webgpu/util/math.ts:422]: nit: shouldn't this be `an` ?

**Review by ben-clayton — APPROVED** [review]

File: src/webgpu/util/math.ts (hunk 1 above)
Comment (zoddicus) [reply src/webgpu/util/math.ts:422]: Done

**Pull request merged**